ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/process/search_tree/Tree.java

```diff
@@ -94,22 +106,32 @@ private Tree() {
                 }
                 currentLeaf.deletePreOptimResultVariant(crac);
             }
+
+            if (hasImproved) {
+                logOptimalLeaf(optimalLeaf, crac);
+            } else {
```

Comment:
Isn't this redundant with the `logOptimalLeaf(optimalLeaf, crac);` (line 95) of the next iteration ?

Reply:
![Capture d’écran de 2020-05-20 14-09-09](https://user-images.githubusercontent.com/53527338/82444498-a19d2a80-9aa3-11ea-9916-c8713aded45b.png)
The idea was to have at one glance, the current optimal leaf, all the generated leaves and the chosen one. So at each search tree iteration you can really understand what happened.

Reply:
So yes it is redundant but I thought it could make the logs clearer

Reply:
ok, it's just one line anyway